ixmp/core.py

```diff
@@ -164,6 +164,48 @@ def scenario_list(self, default=True, model=None, scen=None):
         return pd.DataFrame(self._backend.get_scenarios(default, model, scen),
                             columns=FIELDS['get_scenarios'])
 
+    def export_timeseries_data(self, path, default=True, model=None,
+                               scenario=None, variables=None):
+        """Bulk timeseries export to CSV file.
+
+        Parameters
+        ----------
+        file : os.PathLike
+            File name to export data to; must have the suffix '.csv'.
+
+            Result file will contain the following columns:
```

Comment:
I think we need to move this to a new section of the documentation, which will also contain descriptions of the GDX and Excel file formats.

That can either be in api-backend.rst (named like “File formats”) or in api-python.rst—not sure where is best.

Maybe okay to leave this where it is for now and address that when we bring in the Excel I/O code from message_ix.

Reply:
Fine with moving later.

Reply:
Okay! Please open an issue for it so we don't forget.